### ISSUE TITLE
Stop construction of product 

### ISSUE THREAD

**Issue by bj-rutledge** [bj-rutledge/CRUD-API-Example#44]
The product class should

**Comment by stawuah** [comment]
Hi, @bj-rutledge , if i am getting it right a user can  set a custom price for the product. This change allows for more flexibility in managing product prices, including the ability to set `prices below cost ` if necessary.

**Comment by bj-rutledge** [comment]
Yes, I thought more about the problem and realized I was over complicating it and also limiting the abilities. After reviewing the point of sale system for my business, I found that I can set sell prices below cost, which makes sense to me. Sometimes that's necessary. So, I don't want to limit that.

**Comment by stawuah** [comment]
Okay , I have made some changes but I'd like to ask if there are some pricing strategy before i make a pr

**Comment by bj-rutledge** [comment]
Not really. When running a business, it is sometimes necessary to sell below cost of good. Therefore, I want to ensure that we have the functionality to sell below cog. So no real strategy at this juncture.
